Engine/DataFeeds/FileSystemDataFeed.cs

```diff
@@ -89,53 +81,9 @@ public IEnumerable<Subscription> Subscriptions
             var threadCount = Math.Max(1, Math.Min(4, Environment.ProcessorCount - 3));
             _controller = new ParallelRunnerController(threadCount);
             _controller.Start(_cancellationTokenSource.Token);
-
-            // wire ourselves up to receive notifications when universes are added/removed
-            algorithm.UniverseManager.CollectionChanged += (sender, args) =>
-            {
-                switch (args.Action)
-                {
-                    case NotifyCollectionChangedAction.Add:
-                        foreach (var universe in args.NewItems.OfType<Universe>())
-                        {
-                            var config = universe.Configuration;
-                            var start = _algorithm.UtcTime;
-
-                            var marketHoursDatabase = MarketHoursDatabase.FromDataFolder();
-                            var exchangeHours = marketHoursDatabase.GetExchangeHours(config);
-
-                            Security security;
-                            if (!_algorithm.Securities.TryGetValue(config.Symbol, out security))
-                            {
-                                // create a canonical security object if it doesn't exist
-                                security = new Security(
-                                    exchangeHours,
-                                    config,
-                                    _algorithm.Portfolio.CashBook[CashBook.AccountCurrency],
-                                    SymbolProperties.GetDefault(CashBook.AccountCurrency),
-                                    _algorithm.Portfolio.CashBook
-                                );
-                            }
-
-                            var end = _algorithm.EndDate.ConvertToUtc(_algorithm.TimeZone);
-                            AddSubscription(new SubscriptionRequest(true, universe, security, config, start, end));
-                        }
-                        break;
-
-                    case NotifyCollectionChangedAction.Remove:
-                        foreach (var universe in args.OldItems.OfType<Universe>())
-                        {
-                            RemoveSubscription(universe.Configuration);
-                        }
-                        break;
-
-                    default:
-                        throw new NotImplementedException("The specified action is not implemented: " + args.Action);
-                }
-            };
         }
 
-        private Subscription CreateSubscription(SubscriptionRequest request)
+        private Subscription InternalCreateSubscription(SubscriptionRequest request)
```

Comment:
Maybe instead of `InternalCreateSubscription` this is `CreateDataSubcription` which is different from `CreateUniverseSubscription` ? Just thinking of ways to keep the verbiage between the two options consistent and descriptive (universe vs data subscriptions)

Reply:
Good idea 👍 I'll change it